src/main/java/com/builtbroken/ai/improvements/AIImprovements.java

```diff
@@ -139,7 +138,7 @@ else if (REMOVE_LOOK_IDLE && task.action instanceof EntityAILookIdle)
             }
 
             //Only replace vanilla look helper to avoid overlapping mods
-            if (REPLACE_LOOK_HELPER && (living.getLookHelper() == null || living.getLookHelper().getClass() == EntityLookHelper.class))
```

Comment:
This will break other mods, we need to make sure we pull from the get method and that we only replace EntityLookHelper. As several mods do replace the lookhelper with there own versions using Access transformers, ASM, or reflection.